GetDKAN/dkan
# Using xautoload instead of autoload.

Fixes #2487 

The fact that the tests are actually running (and failing) signals that we are successfully building, which is the only purpose for this PR.

**Manual QA:**
1. Get the code from this branch
2. ahoy dkan remake
3. ahoy dkan reinstall

The expectations are that the site should build and you should be able to go to a resource, configure a datastore and import without issues.
